pegasus/sites.v3/code.org/public/student.haml

```diff
@@ -47,30 +49,38 @@
     background-color: #ffa400;
   }
```

Comment:
Erin is working on the responsiveness of this page right now and she'll remove it in her PR.